src/components/SpaceCreateWarnings.vue

```diff
@@ -37,12 +41,30 @@ const strategySymbolsString = computed(() => {
 
   return `(${symbols.join(', ')})`;
 });
+
+const isAdmin = computed(() => {
+  const admins = (props.space.admins || []).map(admin => admin.toLowerCase());
+  return admins.includes(web3Account.value?.toLowerCase());
+});
 </script>
 
 <template>
   <div class="mb-4 space-y-2">
+    <BaseMessageBlock
+      v-if="space.hibernated"
+      level="warning"
+      is-responsive
+    >
+      This space has been hibernated, and proposals creation has been disabled until the space is reactivated by an admin.
+      <p v-if="isAdmin" class="mt-3">
+      <BaseButton @click="handleReactivateSpace">
```

Comment:
I would use BaseLink instead of a button here, without the `p` element

Reply:
Removed the `p`, isn't a button more appropriate for a CTA ? I also added a "Learn more" link, so it may help to distinguish them